optuna/integration/botorch.py

```diff
@@ -168,7 +190,7 @@ def qei_candidates_func(
     """Quasi MC-based batch Expected Improvement (qEI).
 
     The default value of ``candidates_func`` in :class:`~optuna.integration.BoTorchSampler`
-    with single-objective optimization for constrained problems.
+    with single-objective optimization.
```

Comment:
This content is confusing. I think we should remove this. 

In addition, we need to fix the getter function of the candidates function.